spec/features/tasks/collection_objects/filter_spec.rb

```diff
@@ -214,6 +214,95 @@
           end
         end
       end
+
+      context 'with records specific to users and dates' do
+        let!(:pat) { User.find(1) }
+        let!(:pat_admin) { User.where(name: 'Pat Project Administrator').first }
+        let!(:joe) { FactoryGirl.create(:valid_user) }
+        let!(:prepare) {
+          if $user.blank?
+            $user_id = pat.id
```

Comment:
No.  Not allowed.

Reply:
Try this:
```ruby
let!(:joe) { 
    member = factory_girl_create_for_user(:valid_user, @user) 
    @project.users << member
    member
}
```
But I made do without the global reference.